Automattic/vip-go-mu-plugins
# Add WP-CLI command to validate index post counts

## Description

Adds a new VIP WP-CLI command to validate post type counts between the DB and Elasticsearch index.

Requires ElasticPress to be installed and activated.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Setup Elasticsearch and ElasticPress on test VIP Go site
1. Checkout PR
1. Run `wp vip-es health validate-counts`
1. Verify post counts and command output
